### PR TITLE
Integrated ContentBundle

### PR DESCRIPTION
**Pull request by wouterj**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "require": {
         "php": "^5.5.6|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0"
+        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
+        "sonata-project/admin-bundle": "^3.6.0"
```
Comment (ElectricMaxxx) [inline composer.json]: why an explicit version now? (to understand it)

File: src/DependencyInjection/Factory/ContentAdminFactory.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    {
+        return 'content';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addConfiguration(NodeBuilder $builder)
+    {
+        $builder->scalarNode('admin_class')->defaultValue(StaticContentAdmin::class)->end();
```
Comment (ElectricMaxxx) [inline src/DependencyInjection/Factory/ContentAdminFactory.php]: good point here, do we really like that other way of setting class parameters? had the same on block-bundle (need to replace strings)

Comment (wouterj) [inline src/DependencyInjection/Factory/ContentAdminFactory.php]: Using the `::class` is surely better than putting the FQCN in string (it allows IDE refactorings for instance)

File: composer.json (hunk 1 above)
Comment (wouterj) [inline composer.json]: The ContentBundle's form type relies on some behaviour that's introduced in SonataAdminBundle 3.6.0. If we don't specify this, we seem to also support SonataAdminBundle 3.1.0 (the lowest support version from the dependencies). See also the failing previous builds on this branch.

File: src/DependencyInjection/Factory/ContentAdminFactory.php (hunk 2 above)
Comment (ElectricMaxxx) [inline src/DependencyInjection/Factory/ContentAdminFactory.php]: i know and i really like it. simply forgot it there (fix will follow), bu the question was: that are class parameters, just at an other place. thought they where deprecated.

Comment (ElectricMaxxx) [inline src/DependencyInjection/Factory/ContentAdminFactory.php]: ok

Comment (dbu) [inline src/DependencyInjection/Factory/ContentAdminFactory.php]: => #34

**Pull request merged**